node-aci/build

```diff
@@ -0,0 +1,39 @@
+#! /bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+set -o xtrace
+
+rm -f node.aci
+
+docker2aci docker://debian:jessie
+
+acbuild begin ./library-debian-jessie.aci
+
+acbuild run -- apt-get update
+acbuild run -- apt-get install -y -q apparmor curl iptables
+acbuild run -- apt-get autoremove
+acbuild run -- apt-get clean
+
+acbuild run -- \
+  curl -sSL --fail \
+    "https://get.docker.com/builds/Linux/x86_64/docker-1.11.1.tgz" \
```

Comment:
The community itself doesn't agree on the version of docker to use in kube
1.3, I think this blesses a particular version moving forward.  I prefer
it's removed.  Maybe I just don't understand the expectation for this PR
generally.

On Thursday, June 23, 2016, Mike Danese <notifications@github.com> wrote:

> In node-aci/build
> <https://github.com/kubernetes/kube-deploy/pull/34#discussion_r68332519>:
>
> > +set -o xtrace
> > +
> > +rm -f node.aci
> > +
> > +docker2aci docker://debian:jessie
> > +
> > +acbuild begin ./library-debian-jessie.aci
> > +
> > +acbuild run -- apt-get update
> > +acbuild run -- apt-get install -y -q apparmor curl iptables
> > +acbuild run -- apt-get autoremove
> > +acbuild run -- apt-get clean
> > +
> > +acbuild run -- \
> > +  curl -sSL --fail \
> > +    "https://get.docker.com/builds/Linux/x86_64/docker-1.11.1.tgz" \
>
> Agree with @aaronlevy <https://github.com/aaronlevy> that I am more
> motivated by coupling kubelet/docker versions then making it easy to
> install docker.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kube-deploy/pull/34/files/3173ffd06edce04392130f4d4a9f27458cd33022#r68332519>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8dbMZ0la5he-1RJ21R2EvGIFZuUG7zks5qOxWVgaJpZM4IisM6>
> .
>